src/systemcmds/tests/test_autodeclination.cpp

```diff
@@ -23,7 +23,7 @@ class AutoDeclinationTest : public UnitTest
 
 bool AutoDeclinationTest::autodeclination_check(void)
 {
-	ut_assert("declination differs more than 1 degree", get_mag_declination(47.0, 8.0) - 0.6f < 0.5f);
+	ut_assert("declination differs more than 1 degree", get_mag_declination(47.0, 8.0) - 1.6f < 0.5f);
```

Comment:
The issue is that in the gtest I'm including the `geo_mag_declination.c` file so I have access to the static methods in the file. Because that file is already built in SITL there will be multiple definitions of functions if I include the same file in this test. The only way I see to make my tests work would be to move the function declaration to the .h file, but that doesn't make sense from an encapsulation perspective. Any ideas?